test/NuGet.Clients.Tests/NuGet.VisualStudio.Common.Test/OutputConsoleLoggerTests.ReportError.cs

```diff
@@ -12,11 +14,15 @@ public partial class OutputConsoleLoggerTests
     {
         public class ReportError : LogAndReportErrorTests
         {
+            public ReportError(GlobalServiceProvider sp)
+                : base(sp)
+            { }
+
             [Theory]
             [MemberData(nameof(GetAllMessageVariations))]
-            public void Adds_entry_to_error_list(LogLevel logLevel, int verbosityLevel)
+            public async Task Adds_entry_to_error_list(LogLevel logLevel, int verbosityLevel)
             {
-                VerifyThatEntryToErrorListIsAdded(_outputConsoleLogger.ReportError, logLevel, verbosityLevel);
+                await VerifyThatEntryToErrorListIsAdded(_outputConsoleLogger.ReportError, logLevel, verbosityLevel);
```

Comment:
Since this method is async, should the name be changed to have Async at the end?  I know some people hate it as a kind of "Hungarian Notation", but it's possible to spot async methods being called without waiting when the method name contains the suffix.